server/eclipse-project/src/main/java/us/freeandfair/corla/controller/ContestCounter.java

```diff
@@ -29,13 +29,14 @@ public final class ContestCounter {
   public static final Logger LOGGER =
     LogManager.getLogger(ContestCounter.class);
 
```

Comment:
Elaborate on the comment by saying the constructor has been made private to prevent construction.